modules/administration-guide/pages/installing-prometheus-and-grafana.adoc

```diff
@@ -19,9 +19,18 @@ Alternatively, you can use the link:https://github.com/prometheus-operator/prome
 
 .Procedure
 
-To install Prometheus and Grafana by using `template.yaml`:
+For this example monitoring stack, Prometheus and Grafana will be installed in a new {orch-namespace} called `monitoring`. To install Prometheus and Grafana by using `template.yaml`:
```

Comment:
The first sentence seems redundant to me as I read it because this info is repeated on the next line in the first step.
```suggestion
To install Prometheus and Grafana by using `template.yaml`:
```